src/main/java/org/assertj/core/internal/Strings.java

```diff
@@ -66,6 +66,7 @@
 import java.util.regex.Pattern;
 import java.util.regex.PatternSyntaxException;
 
+import com.sun.istack.internal.NotNull;
```

Comment:
This stray import is causing a compile error on traivs. :(

Reply:
I've been doing Android too long, I'll remove it.

Reply:
Thanks!